### ISSUE TITLE
Create variables while saving the blueprint

### ISSUE THREAD

**Issue by gustavo-hillesheim** [gustavo-hillesheim/scaffolding#13]
Variables are a very useful feature, but currently they aren't easy to set up, the user needs to open each file and replace the text.
For this reason, we should add a option in the saveBlueprint command, to automatically replace some text by a corresponding variable placeholder, the use would something like:

`scaffold saveBlueprint testBlueprint --variable $projectName=src --variable $componentName=test-component`
or
`scaffold saveBlueprint testBlueprint  --variables $projectName=src $componentName=test-component`

Basically, the first part of the value is the variable name (the `$` is optional, this is useful to make explicit that it is a variable), and the second part is where the variable placeholder should be placed. Variables set this way should be placed in file/directory names and content.

**Comment by gustavo-hillesheim** [comment]
Left `$` as required, and did not add option `--variable` since `--variables` is more useful and equally easy to use.
